src/Sarif.Multitool/Rules/SARIF2009.UseConventionalSymbolicNames.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+using Microsoft.Json.Pointer;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool.Rules
+{
+    public class UseConventionalSymbolicNames : SarifValidationSkimmerBase
+    {
+        /// <summary>
+        /// SARIF2009
+        /// </summary>
+        public override string Id => RuleId.UseConventionalSymbolicNames;
```

Comment:
>UseConventionalSymbolicNames [](start = 44, length = 28)

We should drop this rule to a note and change the name to 'ConsiderUsingConventionalNames'. 'Symbolic' isn't buying us much as an adjective (and symbol is a highly used term in analysis itself, so it's a little confusing. We can use 'Consider' for rules that truly are for nice-to-have conventions, as this one is.

Reply:
Just updated! Thank you!

---
In reply to: [445551103](https://github.com/microsoft/sarif-sdk/pull/1938#discussion_r445551103) [](ancestors = 445551103)